lib/resources/tasks/run.js

```diff
@@ -1,22 +1,10 @@
 import gulp from 'gulp';
 import browserSync from 'browser-sync';
 import historyApiFallback from 'connect-history-api-fallback/lib';
+import { CLIOptions } from 'aurelia-cli';
```

Comment:
VS Code again. I'm curious, though, why not use spaces within the {} when it's an import statement? They are within {} everywhere else, so is this some convention of the import statement?